.github/workflows/deploy-polaris-staging.shopifycloud.com.yml

```diff
@@ -0,0 +1,53 @@
+name: Trigger deploy of polaris-staging.shopifycloud.com
+
+on:
+  issue_comment:
+    types:
+      - created
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  trigger-deploy:
+    name: Deploy to staging
+    if: |
+      github.event.issue.pull_request && github.event.comment.body == '/stageit'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get PR SHA
+        id: sha
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+            });
+            return pr.data.head.sha
+      - name: Trigger deploy polaris-staging.shopifycloud.com
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.SHOPIFY_GH_ACCESS_TOKEN }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: "shopify",
+              repo: "polaris-site-container-builder",
+              workflow_id: "trigger-deploy.yml",
+              ref: "staging-bdvt",
```

Comment:
For now, we are not sure how to deploy to staging from the `main` branch of shopify/polaris-site-container-builder, so we're doing it from the `staging-bdvt` branch. Once we've figured that out, we'll update this to be `main`.